### DISCUSSION TITLE
Why can't I use NPM link command for a  package using PNPM

### DISCUSSION THREAD

**Discussion by dongwa** [pnpm/pnpm#4234] [Q&A]
### example
First I use pnpm install the commander for my project, and  use commander create a cli named `pnpmtest`
Then use `npm link` command, 
use `pnpmtest -v` to test this program will get an Error like this image, but if use `pnpm link --global`  command it will work well
Worse still, if publish this to npm.js ,user can not use npm to install it, only use pnpm
please help me,thanks!
![image](https://user-images.githubusercontent.com/38457491/149474874-2ebe91c2-7354-4e12-80d9-2c3b25e44295.png)

error message:
```
dong:~/testCode/pnpmcli$ pnpmtest -v
/usr/local/bin/pnpmtest: line 1: syntax error near unexpected token `('
/usr/local/bin/pnpmtest: line 1: `const { program } = require('commander');'
dong:~/testCode/pnpmcli$ pnpmtest 
/usr/local/bin/pnpmtest: line 1: syntax error near unexpected token `('
/usr/local/bin/pnpmtest: line 1: `const { program } = require('commander');'
```

**Comment by Shpackk** [comment]
Did you get the solution ?

**Reply by Kelmatou** [reply]
@Shpackk You can try `#!/usr/bin/env node` at the top of a js file
